src/coreComponents/constitutive/unitTests/testLinearElasticIsotropic.cpp

```diff
@@ -37,9 +37,9 @@ TEST( LinearElasticIsotropicTests, testAllocation )
   EXPECT_EQ( cm.size(), numElems );
   EXPECT_EQ( cm.numQuadraturePoints(), numQuadraturePoints );
 
-  arrayView1d< real64 const > const & bulkModulus = cm.bulkModulus();
-  arrayView1d< real64 const > const & shearModulus = cm.shearModulus();
-  arrayView3d< real64 const, solid::STRESS_USD > const & stress = cm.getStress();
+  arrayView1d< real64 const > const bulkModulus = cm.bulkModulus().toViewConst();
```

Comment:
is this always required now?

Reply:
The UDC from `arrayView< T >` to `arrayView< T const >` no longer works on rvalues, so for things like this yes it is required. Luckily the error message comes from a `static_assert` and it tells you to just use `toViewConst()`.

See https://github.com/GEOSX/LvArray/pull/211#discussion_r511736953